### DISCUSSION TITLE
renaming OCR-Helper-Scripts

### DISCUSSION THREAD

**Discussion by eroux** [OpenPecha/Housekeeping#5] [Ideas]
I propose we store the various pipelines in OCR-Helper-scripts, that we could rename, perhaps into just `pipelines`

**Comment by 10zinten** [comment]
yeah, but I think it should be more explicit like `bdrc-import-pipelines`. Because, OpenPecha have other sources of text to import from like Tsadra Darma Cloud and it could be named as `tsadra-import-pipelines`, etc.

(Not related to the discussion topic, but I would suggest to have separate repo for each text source import pipeline to have a better development experience.)

**Comment by eroux** [comment]
ah, why not, my experience is that in that kind of situation you end up with a lot of repos that are a bit hard to keep track of... it's what we did for data import at BDRC and I kind of regret it
